hs_composite_resource/models.py

```diff
@@ -32,11 +32,18 @@ def set_default_logical_file(self):
         of the resource that is not already associated with a logical file. """
 
         for res_file in self.files.all():
+            # TODO: This following if block will not be needed once we migrate
+            # raster file type to HStore
```

Comment:
@hyi Look at my related TODO comment in ResourceFile class. The raster file type is still using the generic relation to relate an instance of ResourceFile object to a LogicalFile object. Whereas the nebtcdf file type is using the ForeignKey relation (logical_file_new) to relate an instance of ResourceFile object to a instance of a LogicalFile object. Once we decide that we would store file type metadata as key/value (what I have implemented for nectdf file type in this PR) then the fields in ResourceFile class related GenericRelation with logical file need to be removed.  As a result the above TODO in line 35 needs to be done.